clinvar/src/main/scala/org/broadinstitute/monster/etl/clinvar/models/output/SCVTrait.scala

```diff
@@ -20,6 +20,7 @@ import upack.{Msg, Obj, Str}
   * @param `type` type of the trait
   * @param xrefs stringified JSON objects describing unique IDs for
   *              the trait in databases other than MedGen
+  * @param traitId the ClinVar trait ID of the corresponding VCVTrait, if known
   */
 case class SCVTrait(
   id: String,
```

Comment:
`clinicalAssertionTraitSetId` should go away since we're doing top-down links now